backend/src/services/org/org-service.ts

```diff
@@ -419,24 +435,88 @@ export const orgServiceFactory = ({
   /*
    * Delete organization by id
    * */
-  const deleteOrganizationById = async (
-    userId: string,
-    orgId: string,
-    actorAuthMethod: ActorAuthMethod,
-    actorOrgId: string | undefined
-  ) => {
+  const deleteOrganizationById = async ({
```

Comment:
nit: it is better to keep prs focued on one objective and open seperate ones if focus changes. the shorter a pr is the quicker it can be merged